tensorflow/core/util/tensor_bundle/BUILD

```diff
@@ -78,6 +77,13 @@ cc_library(
     deps = ["//tensorflow/core:lib"],
 )
 
+cc_library(
+    name = "byteswap",
+    srcs = ["byte_swap.cc"],
+    hdrs = ["byte_swap.h"],
+    deps = ["//tensorflow/core:framework"],
```

Comment:
The issue is that this still depends on framework, which is rather big. 

Could we take the pure byte swapping on buffers (without the tensor dependency) and make that its own cc file and target that depends on tensorflow:platform at most? My goal is to get rid of the dependency on framework here.

The XLA interpreter otherwise would depend on building the TensorFlow framework, which is a big dependency.

Reply:
@sherhut Yes, it makes sense. Sorry I've been on vacation for the past few days. I'll make adjustments accordingly and get back to you ASAP. Thanks!

Reply:
Hi @sherhut ,

I've refactored `byte_swap.h` and `byte_swap.cc` to separate target `byteswaparray` from `byteswaptensor`. Now the XLA interpreter would only depend on part of `tensorflow:platform`. 

Could you please take a look? Thank you very much for your advice!